content/en/v3/about/concepts/pipeline-activity.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Pipeline activites
+linktitle: Pipeline activites
+type: docs
+description: Introduction to Jenkins X Pipeline activites
+weight: 20
+aliases:
+  - /v3/about/concepts/pipeline-activites
+---
+
+Jenkins X creates pipeline activites for jobs.
+It's a kubernetes custom resource (CR) which is scoped to a namespace.
+
+Pipeline activities are made up of steps.
+A step can have three kinds
+
+- Stage
+- Preview
+- Promote
+
+Each type of step has a name, status, start and end timestamp.
+A pipeline activity can have steps of different kinds (for example a pipeline activity can have a stage, preview and promote step)
+Let's look at each of these kinds in detail.
+
+### Stage
+
+A stage step is made up of multiple sub steps and is normally associated with the Continuous Integration (CI) part of Jenkins X.
+The integration of Jenkins X with tekton is explained [here](../tekton-integration)
+
```

Comment:
I read it but I think an example would be more helpful, something like this is a stage in pa that is generated from this pr.yaml.

Reply:
Included an example in the deployment lifecycle section.